src/stdlib_io.fypp

```diff
@@ -147,6 +149,7 @@ contains
 
       skiprows_ = max(optval(skiprows, 0), 0)
       max_rows_ = optval(max_rows, -1)
+      delimiter_ = optval(delimiter, " ")
```

Comment:
```suggestion
      delimiter_ = optval(delimiter, delimiter_default)
```

Reply:
@jvdp1 , thanks for your review. The `delimiter_default` constant has been added according to the feedback.